trace/src/lookup.rs

```diff
@@ -431,8 +472,18 @@ impl From<RawRangeTrace> for RawLookupTrace {
 
         let mut counter = 0u64;
 
+        let mut b_ids = Vec::new();
         while counter < value.b {
-            b[0][0].push(
+            if counter as usize % max_height == 0 {
+                b.push(vec![vec![]]);
+                b_ids.push(vec![format!(
```

Comment:
i'm no sure about security of this. Lets leave a comment here to investigate behavior of it